Magento_Theme/requirejs-config.js

```diff
@@ -1,18 +1,19 @@
 var config = {
   map: {
     "*": {
+      "choices"            : "Magento_Theme/js/lib/choices",
       "closestPolyfill"    : "Magento_Theme/js/lib/closest-polyfill",
+      "lazysizes"          : "Magento_Theme/js/lib/lazysizes",
+      "lory"               : "Magento_Theme/js/lib/lory",
+      "noUiSlider"         : "Magento_Theme/js/lib/nouislider.min",
+      "picturefill"        : "Magento_Theme/js/lib/picturefill",
+      "svg4everybody"      : "Magento_Theme/js/lib/svg4everybody",
```

Comment:
Why did you changed oreder of these items?  Should be added alphabetically

Reply:
I grouped them by type then sorted alphabetically, libs are first then rest of scripts. If you don't like it I can switch to previous version